### DISCUSSION TITLE
Obtaining Geoid (EGM2008) to Ellipsoid Proj Pipeline

### DISCUSSION THREAD

**Discussion by scottyhq** [pyproj4/pyproj#1072] [Q&A]
I'm trying to extract a proj4 pipeline ( `projinfo -s EPSG:9518 -t EPSG:7661 -o PROJ --hide-ballpark`) with pyproj via `TransformerGroup(CRS.from_epsg(9518), CRS.from_epsg(7661))` But using TransformerGroup only comes up with a noop transform...  Instead of what `projinfo` reports:

```
Candidate operations found: 1
-------------------------------------
Operation No. 1:

unknown id, Inverse of WGS 84 to EGM2008 height (1) + WGS 84 to WGS 84 (G1150), 3 m, World., at least one grid missing

PROJ string:
+proj=pipeline
  +step +proj=axisswap +order=2,1
  +step +proj=unitconvert +xy_in=deg +xy_out=rad
  +step +proj=vgridshift +grids=us_nga_egm08_25.tif +multiplier=1
  +step +proj=unitconvert +xy_in=rad +xy_out=deg
  +step +proj=axisswap +order=2,1
```

I must be missing something because, the documented example does find grids on the CDN... https://pyproj4.github.io/pyproj/stable/transformation_grids.html#what-grids-to-download 

```
pyproj info:
    pyproj: 3.3.1
      PROJ: 9.0.0
  data dir: /opt/conda/share/proj
user_data_dir: /root/.local/share/proj
PROJ DATA (recommended version): 1.9
PROJ Database: 1.2
EPSG Database: v10.054 [2022-02-13]
ESRI Database: ArcMap 12.9 [2022-02-18]
IGNF Database: 3.1.0 [2019-05-24]

System:
    python: 3.9.10 | packaged by conda-forge | (main, Feb  1 2022, 21:24:11)  [GCC 9.4.0]
executable: /opt/conda/bin/python
   machine: Linux-5.10.104-linuxkit-x86_64-with-glibc2.31

Python deps:
/opt/conda/lib/python3.9/site-packages/_distutils_hack/__init__.py:30: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
   certifi: 2021.10.08
    Cython: None
setuptools: 60.10.0
       pip: 22.0.4
```

**Comment by snowman2** [comment]
Related: https://github.com/OSGeo/PROJ/issues/3177

**Reply by scottyhq** [reply]
Thanks @snowman2 I don't quite follow that linked issue. I found a workaround: `Transformer.from_crs(9518, 7661, allow_ballpark=False).to_proj4()`. Note `allow_ballpark=False` is required. But I don't see anything from debug messages to indicate why the pipeline is not found without `allow_ballpark=False` or how to pass this setting to TransformerGroup?

**Reply by snowman2** [reply]
Thanks for looking into it, I think adding `allow_ballpark` to `TransformerGroup` is something that makes sense to do.

**Reply by snowman2** [reply]
Related to #1073

**Reply by snowman2** [reply]
I think I have a fix in #1075

**Reply by snowman2** [reply]
`allow_ballpark` added in #1076
